public/manifest.json

```diff
@@ -31,16 +31,10 @@
   "background": {
     "scripts": [
       "vendor/async.min.js",
```

Comment:
@DerickMathew Can you remove this library too? This was needed by booking generator only.